src/pages/data_submission/v2/v2-models.tsx

```diff
@@ -312,6 +318,12 @@ export class NihAnvilUse extends StudyProperty {
   }
 }
 
+export type NiHAnvilUseValues
+  = | typeof NihAnvilUse.YES_NHGRI_YES_PHS_ID
+    | typeof NihAnvilUse.YES_NHGRI_NO_PHS_ID
+    | typeof NihAnvilUse.NO_NHGRI_YES_ANVIL
+    | typeof NihAnvilUse.NO_NHGRI_NO_ANVIL
```

Comment:
Just noting an inconsistency with the [schema](https://consent.dsde-dev.broadinstitute.org/#/Schema/getDatasetRegistrationSchemaV1):
```
        "properties": {
          "nihAnvilUse": {
            "type": "string",
            "enum": [
              "I am NHGRI funded and I have a dbGaP PHS ID already",
              "I am NHGRI funded and I do not have a dbGaP PHS ID",
              "I am not NHGRI funded but I am seeking to submit data to AnVIL"
            ]
          }
        }
```
I wonder if the `NO_NHGRI_NO_ANVIL` option would be validated correctly on the back end. It would be nice if we could somehow standardize the models with the schema.